src/xcode/ENA/ENA/Source/Models/Profile/AntigenTestProfile.swift

```diff
@@ -91,5 +91,11 @@ struct AntigenTestProfile: Codable {
 	var city: String?
 	var phoneNumber: String?
 	var email: String?
-
+	
+	var fullName: String? {
+		guard let first = firstName, let last = lastName else {
```

Comment:
Shouldn't the possibility be covered where only firstName or lastName is set?

Reply:
@ArturFriesen this is updated now 👍